controllers/templatesync/template_sync.go

```diff
@@ -389,7 +462,7 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 				}
 
 				// set label to identify parent policy for this template
-				labels[parentPolicyLabel] = instance.GetName()
+				labels[utils.ParentPolicyLabel] = instance.GetName()
 
 				tObjectUnstructured.SetLabels(labels)
 				tObjectUnstructured.SetOwnerReferences([]metav1.OwnerReference{plcOwnerReferences})
```

Comment:
Owner references shouldn't be set on the cluster-scoped policy templates.

Reply:
Okay, I'll update it. I wasn't sure whether we wanted to leave them on for reference.

Reply:
I think they're useful but they are invalid owner references and I don't know if the Kubernetes behavior will change in the future. I think it's safer to not have them.